mne/annotations.py

```diff
@@ -32,7 +32,7 @@ class Annotations(object):
     Parameters
     ----------
     onset : array of float, shape (n_annotations,)
-        Annotation time onsets from the beginning of the recording in seconds.
+        Annotation time onsets relative to the ``orig_time``.
```

Comment:
Maybe say: "Annotation time onsets relative to the ``orig_time``, the starting time of annotation acquisition". The annotation gif is 500 kB, is that okay? You could also perhaps add it directly to `gh-pages` ... ?